lib/Graph/Nodes.cpp

```diff
@@ -519,12 +519,12 @@ bool ChannelwiseQuantizedConvolutionNode::verify() const {
                                getScales().dims().size(), size_t(1), this);
 
   // check qparam sizes
-  isValid &=
-      expectCompareTrue("There must be one filter offset qparam per group",
-                        getOffsets().dims()[0], dim_t(getGroup()), this);
-  isValid &=
-      expectCompareTrue("There must be one filter scale qparam per group",
-                        getScales().dims()[0], dim_t(getGroup()), this);
+  isValid &= expectCompareTrue(
+      "There must be one filter offset qparam per group",
```

Comment:
Fix the error msg. I think since we are not using c2 for cv case, we should just change the impl?

Reply:
Hmmm, maybe disable the specific test for now.